mono/mini/aot-runtime.c

```diff
@@ -2201,6 +2203,8 @@ if (container_assm_name && !container_amodule) {
 		amodule->extra_method_info_offsets = (guint32 *)info->extra_method_info_offsets;
 		amodule->got_info_offsets = info->got_info_offsets;
 		amodule->llvm_got_info_offsets = info->llvm_got_info_offsets;
+		amodule->llvm_got_info_offsets = info->llvm_got_info_offsets;
+		amodule->weak_field_indexes = info->weak_field_indexes;
```

Comment:
copy pasta?

there is `amodule->llvm_got_info_offsets = info->llvm_got_info_offsets;` twice.